### DISCUSSION TITLE
課題14についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#44] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/42#issue-1312894343

↑上記のリンク先にて、質問を書かせていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
> [#42 (comment)](https://github.com/nsc-kiwamu/traning-takeda/pull/42#issue-1312894343)
> 
> ↑上記のリンク先にて、質問を書かせていただいております。 よろしくお願いいたします。

https://github.com/nsc-kiwamu/traning-takeda/pull/42#issuecomment-1192035518
